src/main/java/com/gregtechceu/gtceu/api/data/chemical/material/Material.java

```diff
@@ -907,13 +1029,36 @@ public Builder polymer() {
          * @throws IllegalArgumentException If an {@link PolymerProperty} has already been added to this Material.
          */
         public Builder polymer(int harvestLevel) {
+            return polymer(harvestLevel, 0);
+        }
+
+        /**
+         * Add a {@link PolymerProperty} to this Material.<br>
+         * Will automatically add a {@link DustProperty} to this Material if it does not already have one.
+         *
+         * @param harvestLevel The Harvest Level of this block for Mining.<br>
+         *                     If this Material also has a {@link ToolProperty}, this value will
+         *                     also be used to determine the tool's Mining level.<br>
+         *                     If this Material already had a Harvest Level defined, it will be overridden.
+         * @param burnTime     The Burn Time (in ticks) of this Material as a Furnace Fuel.<br>
+         *                     If this Material already had a Burn Time defined, it will be overridden.
+         * @throws IllegalArgumentException If an {@link PolymerProperty} has already been added to this Material.
+         */
+        public Builder polymer(int harvestLevel, int burnTime) {
```

Comment:
New function to allow specifying a polymer with a burn time similar to other related properties